cmd/root.go

```diff
@@ -147,18 +157,16 @@ func rootCmd(o *options.Options) error {
 		return repoResult.Checks[i].Name < repoResult.Checks[j].Name
 	})
 
-	if o.Format == options.FormatDefault {
-		for checkName := range enabledChecks {
-			fmt.Fprintf(os.Stderr, "Finished [%s]\n", checkName)
-		}
-		fmt.Println("\nRESULTS\n-------")
+	for checkName := range enabledChecks {
+		fmt.Fprintf(os.Stdout, "Finished [%s]\n", checkName)
```

Comment:
these should also be gated behind the default format. and the other line was deleted